### PR TITLE
[16.0] pos_partner_firstname: fix string

### PR DESCRIPTION
**Pull request by alexis-via**
The kind of bad string you only see when working on PO files !

### CODE REVIEW

File: pos_partner_firstname/static/src/js/PartnerDetailsEdit.js [hunk 1]
```diff
@@ -45,7 +45,7 @@ odoo.define("pos_partner_firstname.PartnerDetailsEdit", function (require) {
                         processedChanges.lastname === ""
                     ) {
                         return this.showPopup("ErrorPopup", {
-                            title: _t("Both Customer First and Last Name Are Required"),
+                            title: _t("Both customer first name and last name are required."),
```
Comment (legalsylvain) [inline pos_partner_firstname/static/src/js/PartnerDetailsEdit.js:48]: are you sure on this one ? 

https://www.deepl.com/translator#fr/en/Le%20nom%20et%20le%20pr%C3%A9nom%20du%20client%20sont%20obligatoires

**Review by legalsylvain — COMMENTED** [review]
One remark inline. Otherwise, LGTM.

**Comment by DorianMAG** [comment]
Hi @alexis-via 
This issue https://github.com/OCA/pos/issues/1097 have relation with your PR?

File: pos_partner_firstname/static/src/js/PartnerDetailsEdit.js (hunk 1 above)
Comment (cvinh) [reply pos_partner_firstname/static/src/js/PartnerDetailsEdit.js:48]: This is involved in v17 here https://github.com/OCA/pos/pull/1126

**Pull request closed without merging**